docs/installation/installing-with-npm.md

```diff
@@ -145,7 +155,37 @@ instead use:
 require('govuk-frontend/all')
 ```
 
-#### Polyfills
+### Step 2: Initialise JavaScript
+
+You can initialise all the components with the below snippet:
+
+```js
+  window.GOVUKFrontend.initAll()
+```
+
+You can also initialise individual components using the `data-module` attribute set on the component. If you only have one instance of a component per page, you can follow the below example to initialise it:
+
+```js
```

Comment:
The initialisation is dependent on the include/import method.

- If we use `import Radios from 'govuk-frontend/components/radios/radios'` we assign the exported constructor to the `Radios` variable at import.
- If we include the `all.js` script we need to refer the constructor from the `GOVUKFrontend` global variable `var Radios = window.GOVUKFrontend.Radios`

And then to the initialisation as shown below.